components/binary_sensor/ble_presence.rst

```diff
@@ -55,7 +70,11 @@ address type and advertised name. If you don't see these messages, your device i
 currently not supported.
 
 Please note that devices that show a ``RANDOM`` address type in the logs cannot be used for
-tracking, since their MAC-address periodically changes.
+MAC address based tracking, since their MAC-address periodically changes. Instead you can create
+a BLE beacon, set a unique 16 bit, 32 bit or 128 bit Service UUID and track your device based on that.
+Make sure you don't pick a [GATT Service UUID](https://www.bluetooth.com/specifications/gatt/services/),
```

Comment:
That format is markdown - this page is written in RST - see contributing guide for creating links with RST.